lib/bap_disasm/bap_disasm_rec.ml

```diff
@@ -163,7 +163,8 @@ let ok_nil = function
 
 let is_jump s mem insn =
   Dis.Insn.is insn `May_affect_control_flow ||
-  has_jump (ok_nil (s.lift mem insn))
+  has_jump (ok_nil (s.lift mem insn)) ||
+  Set.mem s.inits (Addr.succ (Memory.max_addr mem))
```

Comment:
given this additional line we need to update the function name to `is_terminator`